cdn-load-balancer/android/ExoPlayerCDNLoadBalancerPlugin/app/src/main/res/xml/network_security_config.xml

```diff
@@ -3,7 +3,6 @@
     <base-config cleartextTrafficPermitted="true">
         <trust-anchors>
             <certificates src="system" />
-            <certificates src="user" />
```

Comment:
Technically, this was there for customers that use a proxy in the middle, so that we accept to SSL handshake with their proxy once they trust it in their phone settings. I guess it's better to remove it and add something to the README, also I'm not full aware of the SSL handshake we're doing (or not) in the C++.